### PR TITLE
Update siril.rb

### PR DESCRIPTION
**Pull request by lock042**
New upstream version

### CODE REVIEW

File: Formula/siril.rb [hunk 1]
```diff
@@ -18,6 +18,7 @@ class Siril < Formula
   depends_on "pkg-config" => :build
   depends_on "adwaita-icon-theme"
   depends_on "cfitsio"
+  depends_on "curl"
```
Comment (fxcoudert) [inline Formula/siril.rb]: macOS already includes curl. Doesn't it work with the system curl anymore?

Comment (lock042) [reply Formula/siril.rb]: Indeed, you probably right.
I'm not a mac user so it is difficult for me to say that.
I remove this dep ?

Comment (fxcoudert) [reply Formula/siril.rb]: Please do

Comment (lock042) [reply Formula/siril.rb]: @fxcoudert : done.

**Pull request closed without merging**